tiledb/sm/array_schema/domain.cc

```diff
@@ -595,12 +586,14 @@ const ByteVecValue& Domain::tile_extent(unsigned i) const {
   return dimension_ptrs_[i]->tile_extent();
 }
 
-std::vector<ByteVecValue> Domain::tile_extents() const {
-  std::vector<ByteVecValue> ret(dim_num_);
-  for (unsigned d = 0; d < dim_num_; ++d)
+tdb::pmr::vector<ByteVecValue> Domain::tile_extents() const {
+  tdb::pmr::vector<ByteVecValue> ret(
+      dim_num_, memory_tracker_->get_resource(MemoryType::DOMAINS));
```

Comment:
Oooh, I think this isn't gonna fly. We're returning a vector of unknown lifetime that would require a lifetime shorter than this Domain instance. Given that both cases are a copy, I'd just avoid the PMR return value. In the future we can figure out a better way to handle this sort of case.

Reply:
Hurms. Let me know if that wreaks havoc on those test cases. If it gets to be super annoying we can take a closer look at doing this differently.

Reply:
Done